inference-engine/src/transformations/src/transformations/common_optimizations/hswish_fusion.cpp

```diff
@@ -142,3 +142,48 @@ ngraph::pass::HSwishFusionWithHSigmoid::HSwishFusionWithHSigmoid() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(mul_pattern, matcher_name);
     register_matcher(m, callback);
 }
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSwishFusionWithClampMul, "HSwishFusionWithClampMul", 0);
+
+ngraph::pass::HSwishFusionWithClampMul::HSwishFusionWithClampMul() {
+    MATCHER_SCOPE(HSwishFusionWithClampMul);
+    // Replaces a sub-graph (Clamp(x + 3, 0, 6) * x) with a HSwish * 6.
+    const auto input = ngraph::pattern::any_input();
+    const auto add_constant = ngraph::pattern::wrap_type<ngraph::opset7::Constant>();
+    const auto add = ngraph::pattern::wrap_type<ngraph::opset7::Add>({input, add_constant});
+    const auto clamp = ngraph::pattern::wrap_type<ngraph::op::v0::Clamp>({add});
+    const auto mul = ngraph::pattern::wrap_type<ngraph::opset7::Multiply>({clamp, input});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher &m) {
+        const auto &pattern_to_output = m.get_pattern_value_map();
+        const auto x_output = pattern_to_output.at(input);
+
+        const auto add_const_value = std::dynamic_pointer_cast<ngraph::opset7::Constant>(pattern_to_output.at(add_constant).get_node_shared_ptr());
+
+        const bool valid_constant_values = op::util::has_constant_value(add_const_value, 3.0);
+
+        if (!valid_constant_values) {
+            return false;
+        }
```

Comment:
```suggestion
        const auto add_const_value = std::dynamic_pointer_cast<ngraph::opset7::Constant>(pattern_to_output.at(add_constant).get_node_shared_ptr());
        if (!op::util::has_constant_value(add_const_value, 3.0)) {
            return false;
        }
```

Reply:
fix in bbe29c230e7d5410e4f9c99dcc72e942bf675461